### DISCUSSION TITLE
Implementing the VF2++ in GSoC

### DISCUSSION THREAD

**Discussion by mreza-kiani** [networkx/networkx#5462] [General]
Hi,
I'm Mohammad Reza Kianifar, and I'm interested in your project for GSoC. I study Master of Computer Science at the University of Calgary and my thesis is about graph generalization, and I'm really interested in graph-based problems. I was wondering can I do anything before applying for this project to increase my chance of getting it?
Thanks for your help in advance.
@dschult  @MridulS  @boothby

**Comment by MridulS** [comment]
Hi @mreza-kiani!

Thanks for your interest in contributing to NetworkX!

Usually we like to see some prior contribution to NetworkX (docs/easy issues) or any other scientific python open source software. As a first time you can go through our Contribution guide (https://networkx.org/documentation/latest/developer/contribute.html)

**Comment by mreza-kiani** [comment]
Hi @MridulS 
Thanks for your response.

Do you have any suggested issue from your repository that is still open and I can work on it? Because I looked at NetworkX issues but I couldn't find any suitable issue for myself (someone new to the source code) to start working on.
